# HG changeset 9739 patch
From: kaf24@firebug.cl.cam.ac.uk
# Node ID 36c25ed503e7df95ea55cdf8e1af8b2ceb142be5
# Parent  3f599940ba6ee44bf01522b450b7b569dbf112b3
Subject: Export some tlb-flush functions to modules,
which are directly accessible via macros in tlbflush.h.
Signed-off-by: Keir Fraser <keir@xensource.com>
xen-unstable changeset:   10334:ac5e98c1c46692083ca76a1cb4ad66c5bc37e9d5
xen-unstable date:        Tue Jun 13 14:08:37 2006 +0100

Acked-by: Jan Beulich <jbeulich@novell.com>

Index: head-2006-06-13/arch/i386/mm/hypervisor.c
===================================================================
--- head-2006-06-13.orig/arch/i386/mm/hypervisor.c	2006-06-13 08:35:17.000000000 +0200
+++ head-2006-06-13/arch/i386/mm/hypervisor.c	2006-06-16 10:57:12.000000000 +0200
@@ -133,6 +133,7 @@ void xen_tlb_flush(void)
 	op.cmd = MMUEXT_TLB_FLUSH_LOCAL;
 	BUG_ON(HYPERVISOR_mmuext_op(&op, 1, NULL, DOMID_SELF) < 0);
 }
+EXPORT_SYMBOL(xen_tlb_flush);
 
 void xen_invlpg(unsigned long ptr)
 {
@@ -141,6 +142,7 @@ void xen_invlpg(unsigned long ptr)
 	op.arg1.linear_addr = ptr & PAGE_MASK;
 	BUG_ON(HYPERVISOR_mmuext_op(&op, 1, NULL, DOMID_SELF) < 0);
 }
+EXPORT_SYMBOL(xen_invlpg);
 
 #ifdef CONFIG_SMP
 
